docker-for-mac/faqs.md

```diff
@@ -154,6 +154,13 @@ response.
 If you do not want auto-send of usage data, use the Stable channel. For more
 information, see [Stable and Edge channels](#stable-and-edge-channels) ("What is the difference between the Stable and Edge versions of Docker for Mac?").
 
+### How does Docker Desktop handle Personal Data?
```

Comment:
nitpicking, but maybe: "How is Personal Data handled in Docker Desktop?"

Reply:
Only capitalize the first word and product names. So it should be "How does Docker Desktop handle personal data?"

Reply:
"How is personal data handled in Docker Desktop?"